UI/window-basic-main.cpp

```diff
@@ -11053,6 +11053,33 @@ QColor OBSBasic::GetSelectionColor() const
 	}
 }
 
+const std::vector<MultitrackVideoViewInfo> &
+OBSBasic::GetAdditionalMultitrackVideoViews()
```

Comment:
This does nothing but return a copy of the value. Probably fine to fetch the value directly?

Reply:
not sure I follow, this returns a const reference, rather than a copy (i.e. you're allowed to read entries, but not modify them)

Reply:
@tt2468 Could you clarify the concern here?